File: Section Reports/Data/LINQ/C#/LINQ/ViewerForm.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Windows.Forms;
namespace GrapeCity.ActiveReports.Samples.LINQ
{
	public partial class ViewerForm : Form
	{
		public ViewerForm()
		{
			InitializeComponent();
		}
	  
		
		// Define a structure for LINQtoObject.
		private void ViewerForm_Load(object sender, EventArgs e)
		{
			// To generate a report.
			rptLINQtoObject rpt = new rptLINQtoObject();
		   
			// To run the report.
			arvMain.LoadDocument(rpt);
		}
	}
}

File: Section Reports/Data/LINQ/VB.NET/LINQ/ViewerForm.vb
Public Class ViewerForm
	'  LINQ to Object
	Private Sub ViewerForm_Load(ByVal sender As System.Object, ByVal e As EventArgs) Handles MyBase.Load
		' To generate a report.
		Dim rpt As New rptLINQtoObject
		arvMain.LoadDocument(rpt)
	End Sub
End Class
